pkg/api/testimagestreamtagimport/v1/types.go

```diff
@@ -31,12 +31,13 @@ const (
 
 // SetImageStreamLabels sets namespace and name labels so we can easily
 // filter for specific imports
-func (t *TestImageStreamTagImport) SetImageStreamLabels() {
+func (t *TestImageStreamTagImport) SetImageStreamLabels() *TestImageStreamTagImport {
```

Comment:
While I see what you did in the test code with this, a setter that returns a reference to the object itself is definitely surprising code ...

Reply:
I've renamed it to `WithImageStreamLabels` to clarify the semantics, WDYT?

Reply:
FWIW the `sets` from apimachinery also return themselves in their setter: https://github.com/openshift/ci-tools/blob/fa75e5c8447f70771bdf3346b6cfc0bec9875218/vendor/k8s.io/apimachinery/pkg/util/sets/string.go#L49-L54

Reply:
Wow, I had no idea.